utils/service/txmgr/mocks/TxManager.go

```diff
@@ -0,0 +1,77 @@
+// Code generated by mockery v2.26.0. DO NOT EDIT.
+
+package mocks
+
+import (
+	context "context"
+
+	common "github.com/ethereum/go-ethereum/common"
+
+	mock "github.com/stretchr/testify/mock"
+
+	txmgr "github.com/kroma-network/kroma/utils/service/txmgr"
+
+	types "github.com/ethereum/go-ethereum/core/types"
```

Comment:
Please put together except for kroma one!

Reply:
This is generated code by mockery, should I also edit this file?

Reply:
Ah okay, in that case, we need to consider it exceptional!